README.md

```diff
@@ -175,36 +179,16 @@ Creating and cloning a build:
       "container_name": "base",
       "config_file": "lxc.config",
       "init_timeout": "120s",
-      "template_parameters": [
-        "-d",
-        "ubuntu",
-        "-r",
-        "trusty",
-        "-a",
-        "amd64"
-      ],
-      "template_name": "ubuntu",
-      "cleanup_first": true
+      "rootfs": {
+        "archive": "/path/to/rootfs.tar.gz",
+        "config": "/path/to/lxc.config"
```

Comment:
Yah, this is a good question. I don't have a full answer for you yet. Right now, the packer build will include a config file along side the `rootfs.tar.gz`. My high level idea is that the output of this plugin should be usable, as-is, as its input as well.